## Rule's description is here (gridsome/require-g-image-src)

## :book: Rule Details

:-1: Examples of **incorrect** code for this rule:

```html
Bad example
```

:+1: Examples of **correct** code for this rule:

```html
Good example
```

## :wrench: Options

If this rule has option, write here.
